cpp/tests/utilities/column_utilities.cu

```diff
@@ -804,6 +805,12 @@ bool expect_column_properties_equal(column_view const& lhs,
                                     column_view const& rhs,
                                     debug_output_level verbosity)
 {
+  if (cudf::detail::has_nonempty_nulls(lhs, cudf::get_default_stream())) {
+    throw std::invalid_argument("Unsanitized lhs column");
```

Comment:
Let's change the message to something like `"lhs column has non-empty nulls"` so similar, as `"unsanitized"` is not directly stating the issue.

Reply:
Sure @ttnghia, thanks for the feedback!